src/linalg.h

```diff
@@ -31,12 +42,15 @@ template<typename Derived> bool is_nan(const Eigen::MatrixBase<Derived>& x) {
 
 BoundingBox operator*(const Transform3d &m, const BoundingBox &box);
 
-class Color4f : public Eigen::Vector4f
+// Vector4f is fixed-size vectorizable
+// Use Eigen::DontAlign so we can store Color4f in STL containers
+// https://eigen.tuxfamily.org/dox/group__DenseMatrixManipulation__Alignement.html
+class Color4f : public Eigen::Matrix<float, 4, 1, Eigen::DontAlign>
```

Comment:
Color4f was another issue with Eigen alignment but in this case I decided it was simpler and easier to specifically set `Eigen::DontAlign` since colors are not heavily computed on.